hello/README.md

```diff
@@ -0,0 +1,26 @@
+It is simple hello world gWASM program. Download the `hello` directory, enter it, update paths in `task.json` file
+```
+        "input_dir": "/home/lukaszglen/wasm_test_5/hello/in",
+        "output_dir": "/home/lukaszglen/wasm_test_5/hello/out",
```

Comment:
I think it'd be prudent not to include hardcoded paths to your config. What do you think?